policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+# Project Director Election Process
+
+## Aims
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria, focusing on shared power and new leadership rather than solely prioritizing the best candidate.
```

Comment:
Do we have an "outlined candidate criteria"? I would have expected a section entitled "candidate criteria" that lists what a "good" candidate for this role would be.

Reply:
The use of the word "outlined" makes it explicit that the criteria is written and should be explicitly referred to. I think we should leave that as ambiguous as which criteria we're referring to. If we want that criteria to live in the role description document we should say that: 

> * Select candidates who fulfill the outlined candidate criteria in the [role description document](TODO: link), focusing on shared power and new leadership rather than solely prioritizing the best candidate.

Reply:
```suggestion
* Select candidates who fulfill the outlined candidate criteria in the [role description document](TODO: link), focusing on shared power and new leadership rather than solely prioritizing the best candidate.
```